src/_scss/layouts/default/stickyHeader/header.scss

```diff
@@ -1,26 +1,26 @@
 $search-header-height: rem(66);
 
-.search-header-wrapper {
+.sticky-header__wrapper {
     position: relative;
     height: $search-header-height;
 }
 
-.search-header-container {
+.sticky-header__container {
     width: 100%;
     background-color: #4A4A4A;
     color: $color-white;
     // bottom shadow cast on the content
     box-shadow: 0 2px 2px rgba(0,0,0,.3);
 
-    &.sticky {
+    &.sticky-header__container_sticky {
         @include media($medium-screen) {
             position: fixed;
             top: 0;
             z-index: $z-header;
         }
     }
 
-    .search-header {
+    .sticky-header {
         @import "mixins/fullSectionWrap";
         @include fullSectionWrap(0, 0);
```

Comment:
We can't put `fullSectionWrap` at the block level because it is a mixin that primarily applies margin, which is [disallowed](https://en.bem.info/methodology/quick-start/#block).